tidy3d/web/api/container.py

```diff
@@ -191,9 +196,9 @@ def run(self, path: str = DEFAULT_DATA_PATH) -> SimulationDataType:
         return self.load(path=path)
 
     @cached_property
-    def _task_id(self):
-        if self.task_id:
-            return self.task_id
+    def task_id(self):
```

Comment:
add the return for this since it changed to return self.upload()

Reply:
ah yes, I forgot (I haven't done a full lint check yet, just basics). Thanks